invenio_app_ils/circulation/stats/views.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio App ILS circulation stats views."""
+
+from __future__ import absolute_import, print_function
+
+from datetime import datetime
+
+from flask import Blueprint, request
+from invenio_pidstore import current_pidstore
+from invenio_records_rest.utils import obj_or_import_string
+from invenio_rest import ContentNegotiatedMethodView
+
+from invenio_app_ils.circulation.stats.api import fetch_most_loaned_documents
+from invenio_app_ils.errors import InvalidParameterError
+from invenio_app_ils.pidstore.pids import DOCUMENT_PID_FETCHER, \
+    DOCUMENT_PID_TYPE
+
+
+def create_most_loaned_documents_view(blueprint, app):
+    """Add url rule for most loaned documents."""
+    endpoints = app.config.get("RECORDS_REST_ENDPOINTS", [])
+    document_endpoint = endpoints.get(DOCUMENT_PID_TYPE, {})
+
+    search_serializers = document_endpoint.get("search_serializers")
+    serializers = {
+        mime: obj_or_import_string(func)
+        for mime, func in search_serializers.items()
+    }
+
+    view_class = MostLoanedDocumentsResource
+
+    blueprint.add_url_rule(
+        "/circulation/stats/most-loaned",
+        view_func=view_class.as_view(
+            view_class.view_name,
+            serializers=serializers
+        ),
+        methods=["GET"]
+    )
+
+
+def create_stats_blueprint(app):
+    """Add statistics views to the blueprint."""
+    blueprint = Blueprint("invenio_app_ils_stats", __name__, url_prefix="")
+
+    create_most_loaned_documents_view(blueprint, app)
+
+    return blueprint
+
+
+class MostLoanedDocumentsResource(ContentNegotiatedMethodView):
+    """Statistics view for the documents with the most loans."""
+
+    view_name = "most-loaned"
+    bucket_size = 30
+
+    def _validate_bucket_size(self):
+        """Validate bucket size parameter."""
+        size_param = request.args.get("size", self.bucket_size)
+        try:
+            return int(size_param)
```

Comment:
Last thing I promise, should we validate that the int is not greater than a number? e.g. 10000?